ci-operator/jobs/openshift/verification-tests/openshift-verification-tests-master-periodics.yaml

```diff
@@ -91,7 +91,7 @@ periodics:
     ci.openshift.io/generator: prowgen
     job-release: "4.10"
     pj-rehearse.openshift.io/can-be-rehearsed: "true"
-  name: periodic-ci-openshift-verification-tests-master-installer-rehearse-installer-rehearse-aws-disconnected-private-usgov
+  name: periodic-ci-openshift-verification-tests-master-installer-rehearse-installer-rehearse-aws-private-usgov
```

Comment:
same as https://github.com/openshift/release/pull/27656/files?diff=split&w=0#r854938770, delete lines from 77 to 151.
https://github.com/openshift/release/blob/78a81a20b3e3b4085b3fc1c2a52944b5b23306e3/ci-operator/jobs/openshift/verification-tests/openshift-verification-tests-master-periodics.yaml#L77-L151

Reply:
Done